### PR TITLE
Clean out server files

### PR DESCRIPTION
**Pull request by h0h0h0**
* #35 - remove flask deps and files
* #44 - remove aws bucket save
* Remove dockerfile server ref
* setup.py remove web script
* requirements.txt cleaned up
* precommit yaml filename fixes

Files have been extracted to new project; shaun will share with team.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -49,20 +50,24 @@ python-dateutil==2.8.0
 python-dotenv==0.10.3
 python-pptx==0.6.18
 pyunsplash==1.0.0b8
+PyYAML==5.1.2
```
Comment (twinters) [inline requirements.txt]: Why are there new requirements? :o

**Comment by korymath** [comment]
LGTM. Thomas, mind taking the lead on merge once your questions are sorted?

File: requirements.txt (hunk 1 above)
Comment (h0h0h0) [reply requirements.txt]: let me give it a good cleaning. i removed and recreated requirements so an updated library might have brought in some dependencies.

Comment (h0h0h0) [reply requirements.txt]: I think this is resolved now.

Comment (twinters) [reply requirements.txt]: Superb! Looks good to me! :D Thanks!

**Pull request merged**